src/config/topics.js

```diff
@@ -437,10 +439,20 @@ export const railPlus = {
   exportConfig: {
     publishedAt: '11/2023',
     year: '2023',
-    overlayImageUrl: railplusLegend,
-    exportFileName: `RAILplus Streckennetz ${new Date()
-      .toISOString()
-      .slice(0, 10)}`,
+    overlayImageUrl: (i18n) => {
```

Comment:
just pass the language we don't need all the object.

Reply:
ok